src/main/java/org/sagebionetworks/web/client/view/TeamViewImpl.java

```diff
@@ -159,6 +159,7 @@ public void clear() {
 		deleteTeamItem.setVisible(false);
 		leaveTeamItem.setVisible(false);
 		publicJoinField.setVisible(false);
+		toolsMenu.setVisible(false);
```

Comment:
During view reset, hide the Team Actions menu button (so by default this is not visible).